tasks/buildah.yaml

```
@@ -79,8 +79,8 @@ spec:
     name: digest-to-results
     resources: {}
     script: |
-      cat "$(workspaces.source.path)"/image-digest | tee $(results.IMAGE_DIGEST.path)
-      echo "$(params.IMAGE)" | tee $(results.IMAGE_URL.path)
+      cat $(workspaces.source.path)/image-digest | tee $(results.IMAGE_DIGEST.path)
+      echo $(params.IMAGE) | tee $(results.IMAGE_URL.path)
```

```suggestion
      cat "$(workspaces.source.path)"/image-digest | tee $(results.IMAGE_DIGEST.path)
      echo "$(params.IMAGE)" | tee $(results.IMAGE_URL.path)
```
I think we should keep the quotes, it's safer with weird paths/parameters. In a complementary PR #24 the quotes were added, I think we should be consistent with that here.

---

Any tie-breaker votes on the quotes? :)

---

I'm okay either way, I was thinking that we'd like to keep them as #24 introduced them there and it is a bit safer (say someone passes `IMAGE=*`). Looking at it again, it is not quoted for `buildah`, but I'm guessing buildah doesn't expand globs on it's own, so indeed the point about `buildah` failing first stands.

---

Yeah works for me.